### PR TITLE
[ parser ] Allow a simple PEEK without a slice.

### LINKED ISSUE #7: PEEK does not work
Straight from the book right above the [cheat sheet](https://pest.rs/book/grammars/syntax.html#cheat-sheet):
```
raw_string_interior = {
    (
        !("\"" ~ PEEK)    // unless the next character is a quotation mark
                          // followed by the correct amount of number signs,
        ~ ANY             // consume one character
    )*
}
```

The `PEEK` causes a `OPENING_BRACK expected, got ')'`.

The current BNF only accepts `peek_slice ::= PEEK_TOKEN OPENING_BRACK integer? RANGE_OPERATOR integer? CLOSING_BRACK`.

### PR DESCRIPTION
**Pull request by MalteSchledjewski**
Factor out the slice part into a rule of its own and make it optional in the PEEK rule.

This fixes #7 for me.

### CODE REVIEW

**Comment by MalteSchledjewski** [comment]
I have no experience in developing these plug-ins.  
You should check whether the the `extends`, `pins`, etc. make sense.

**Comment by ice1000** [comment]
This change is not good enough, I'll refactor and merge tonight.
Thanks for your contribution!

File: grammar/pest.bnf [hunk 1]
```diff
@@ -77,10 +77,16 @@ push       ::= PUSH_TOKEN OPENING_PAREN expression CLOSING_PAREN {
...
-peek_slice ::= PEEK_TOKEN OPENING_BRACK integer? RANGE_OPERATOR integer? CLOSING_BRACK {
+peek ::= PEEK_TOKEN peek_slice?{
 	extends=expression
 	pin=1
-	name="PEEK slice command"
+	name="PEEK command"
+}
+
+peek_slice ::= OPENING_BRACK integer? RANGE_OPERATOR integer? CLOSING_BRACK {
+	extends=expression
```
Comment (ice1000) [inline grammar/pest.bnf]: This shouldn't be an expression

**Comment by ice1000** [comment]
Merged, your commit is [here](https://github.com/pest-parser/intellij-pest/commit/17925deb7446f58152740dd2b52bdab1d522fa37).

**Pull request closed without merging**